Updates for Kim (#56)
mulesoft/docs-partner-manager: modules/ROOT/pages/configure-host.adoc

> 

Reviewer: HIDDEN
No edits.

= Configure Host
ifndef::env-site,env-github[]
include::_attributes.adoc[]
endif::[]

> Your organization or your business group is considered the host partner in Anypoint Partner Manager.

Reviewer: ```suggestion
Your organization or your business group is considered the _host partner_ in Anypoint Partner Manager.
```

> In the Anypoint platform, your hostname is the same name as your Anypoint Platform business group.

Reviewer: ```suggestion
In Anypoint Platform, your _hostname_ is the same name as the name of your Anypoint Platform business group.
```

> An Anypoint Platform business group is a self-contained resource that was set up to hold Anypoint Platform resources such as APIs and applications.

Reviewer: ```suggestion
An _Anypoint Platform business group_ is a self-contained resource that holds Anypoint Platform resources such as APIs and applications.
```

Reviewer: So, the previous texts have been about defining host and hostname and business group.... How does that lead to the following text about the host page?

Reviewer: I'm struggling with the way that this topic is trying to evolve from definitions to the actual job of configuring the host. It seems to me that we could do this in a much simpler way, something like...:

"Your first task in using Anypoint Partner Manager is to configure your business organization as the _host_ partner in your B2B relationship with your _trading partners_. To accomplish this, follow these steps:"

...and go straight into the steps of the subtopic, without including the rest of this intro or the subheading.

Within the host page you can do the following:

* Add identifiers to uniquely identify yourself.

* Add digital certificates associated to your organization.

* Configure details associated to the Payload Storage API (the same one you configured as part of the the Setup instructions).
The storage API is a mandatory configuration where your B2B message payloads are stored.

== Update the Host's Profile

> 

Reviewer: OMIT...because all of this is detailed next.

. Open Anypoint Partner Manager and select the name of your organization from the left navigation pane.
You can configure the your indentifiers, certificates, and the location and configuration of your payload storage.

> 

Reviewer: OMIT, because the next steps cover this.

> . Click *New* within the *Identifiers* section to begin adding an identifier related to your organization.

Reviewer: ```suggestion
. In the Identifiers section, click *New*.
```

> . Select an identifier type from the *Type* drop-down menu. This uniquely identifies you.

Reviewer: ```suggestion
. From the Type menu, select a type that uniquely identifies you.
```

> For example, depending on your setup, select one of the following: AS2, DUNS number, X12 GS, or X12-ISA.

Reviewer: OMIT, because this is self-evident in the UI.

> . Provide a value for your identifier (and qualifier for X12-ISA) and click *Create identifier* to save your changes. See xref:x12-identity-settings.adoc[X12 - ISA Identifier Values] for more information on X12-ISA settings.

Reviewer: ```suggestion
. Provide a value for your identifier type (and a qualifier, if you choose the X12-ISA type) and click *Create identifier*. 
See xref:x12-identity-settings.adoc[X12-ISA Identifier Values] for more information about X12-ISA settings.
```
Consider omitting the cross-ref. You typically do not want them in Task steps.

> . Click *New* within the *Certificates* section to begin to upload any needed digital certificates (such as an X.509 certificate in PKCS12 file format for the AS2 transport protocol).

Reviewer: ```suggestion
. In the Certificates section, click *New* to upload any necessary digital certificates (such as an X.509 certificate in PKCS12 file format for the AS2 transport protocol).
```
Consider ending this step after either *New* or "certificates."

> 

Reviewer: HIDDEN
Edited; please compare with original:

. Click *Choose file* to select the certificate, typically a `.PEM` file).
. Click *Upload certificate*.
. Optionally, click *Edit* within the *Payload storage* section to change any of the existing payload storage information (for example, URL location, timeout settings, and acknowledgments).

After configuring your business organization as host, you next configure your B2B trading partners.

== See Also
* xref:x12-identity-settings.adoc[X12-ISA Identifier Values]
* xref:x12-receive-read-settings.adoc[X12 Settings - Receive from Partner]

Author: STET for this release.
